Nodejs/Product/TargetsVsix/Microsoft.NodejsToolsV2.targets

```diff
@@ -0,0 +1,313 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
```

Comment:
Is "14.0" the appropriate value if this will only be used with new projects?

Reply:
Fixed.